backend/controller/sql/queries.sql

```diff
@@ -731,3 +732,9 @@ UPDATE topic_subscriptions
 SET state = 'idle'
 WHERE name = @name::TEXT
       AND module_id = (SELECT id FROM module);
+
+-- name: GetTopicByKey :one
+SELECT *
```

Comment:
nit: I wonder if it makes sense to just return the topic name as part of `GetSubscriptionsNeedingUpdate`

Reply:
ooooo nice catch! done

Reply:
Turns out it's already part of the topic key payload. Thanks for catching that! Now I can revert all the sql changes :)